gapis/trace/android/trace.go

```diff
@@ -133,7 +133,9 @@ func (t *androidTracer) ProcessProfilingData(ctx context.Context, buffer *bytes.
 		return nil, log.Err(ctx, err, "Failed to read trace buffer")
 	}
 	processor, err := perfetto.NewProcessor(ctx, rawData)
-	defer processor.Close()
+	if processor != nil {
+		defer processor.Close()
+	}
```

Comment:
To simplify this, just have the `processor.Close` function handle a nil receiver object.

Reply:
Thanks for suggestion, done! (Finally I see a nice thing with golang :))